extensions/amp-story/1.0/test/validator-amp-story-amp-experiment.out

```diff
@@ -0,0 +1,82 @@
+FAIL
```

Comment:
We name the `FAIL` test with the suffix `*-error.html` and the `PASS` test without the suffix. 

Could you add a passing test and rename these two files with the `-error` suffix please?